packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -560,7 +560,9 @@ export const provideAndStartDirector = (...args) => {
   const [baggage] = args;
 
   const director = provide(baggage, 'director', makeVaultDirector);
-  director.helper.start();
+  director.helper
+    .start()
+    .catch(err => console.error('🚨 vaultDirector failed to start:', err));
```

Comment:
it's caught so it's not void. it will change log output from `🤞 UnhandledRejectionWarning` to the siren. That's appropriate because this has known (high) severity